app/src/main/java/ch/epfl/sdp/blindly/chat/Message.kt

```diff
@@ -0,0 +1,28 @@
+package ch.epfl.sdp.blindly.chat
+
+class Message {
+
+    /**
+     * Empty constructor is needed for the Realtime Database.
+     */
+    constructor()
+
+    // timestamp is to achieve correct ordering of the messages and ensure fetching is in
+    // same ordering the next time users open the chat
+    // Note: I considered usage from different timezones however since matches are only
+    // allowed to be within a certain distance, there won't be any problem.
```

Comment:
What if both matches are close but on the limit of 2 timezones ?

Reply:
I'm willing to take the punishment if that happens :D Though while thinking about the problem I wonder what would happen if timezone is changed manually by hand. I will look into this.

Reply:
Javadoc was unfortunately not very clear but upon further research I learnt that currentTimeMillis() is not timezone dependent as it gets the number of milliseconds passed since January 1st 1970 UTC. Noting that it's still local clock dependent so there could possibly be changes due to local clock set incorrectly but I think it's a bit too much nitpicking at this point.

Reply:
Yeah it is, I guess it's really not that important